exporter/collector/integrationtest/testcases/testcases_metrics.go

```diff
@@ -37,22 +37,35 @@ var MetricsTestCases = []TestCase{
 		ExpectFixturePath:    "testdata/fixtures/metrics/counter_expect.json",
 		ConfigureCollector: func(cfg *collector.Config) {
 			cfg.MetricConfig.InstrumentationLibraryLabels = true
+			cfg.MetricConfig.ServiceResourceLabels = false
```

Comment:
could you add a comment explaining why resource labels are disabled? I know you explained it to me, but it's already fuzzy

Reply:
added a comment